src/components/Solutions/EvaluationDetail/EvaluationDetail.js

```diff
@@ -155,6 +157,22 @@ const EvaluationDetail = ({
               </em>
             </td>
           </tr>
+
+          {viewResumbissions &&
```

Comment:
`viewResubmissions` means that I can see one icon if this submission is in debug mode?

Reply:
If the user may not see the resubmissions (e.g., a student), the user also does not now about debug-mode resubmits.